rts/Sim/Units/Scripts/UnitScript.cpp

```diff
@@ -573,7 +573,11 @@ bool CUnitScript::EmitAbsSFX(int sfxType, const float3& absPos, const float3& ab
 				CWeapon* w = unit->weapons[index];
 
 				const SWeaponTarget origTarget = w->GetCurrentTarget();
-				const SWeaponTarget emitTarget = {absPos + absDir};
+
+				// the following used to be absPos + absDir, which
```

Comment:
ditto, "this was <incorrect value>" doesn't sound helpful.

on the other hand: won't this cause problems if the target point ends up underground or out of range (in case we're a ballistic weapon so may not be able to reach nominal range due to gravity)?

Reply:
Yeah this could be a valid concern, I'll probably revert to how it was.